### PR TITLE
[zh] Update content/zh/docs/reference/_index.md

### PR DESCRIPTION
**Pull request by my-git9**
Update content/zh/docs/reference/_index.md

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/assign @tengqm

File: content/zh/docs/reference/_index.md [hunk 1]
```diff
@@ -144,14 +154,32 @@ operator to use or manage a cluster.
...
 
-* [kubelet 配置 (v1beta1)](/zh/docs/reference/config-api/kubelet-config.v1beta1/)
-* [kube-scheduler 配置 (v1beta1)](/zh/docs/reference/config-api/kube-scheduler-config.v1beta1/)
-* [kube-scheduler 策略参考 (v1)](/zh/docs/reference/config-api/kube-scheduler-policy-config.v1/)
+* [kube-apiserver 配置 (v1alpha1)](/zh/docs/reference/config-api/apiserver-config.v1alpha1/)
+* [kube-apiserver 配置 (v1)](/zh/docs/reference/config-api/apiserver-config.v1/)
+* [kube-apiserver 加密 (v1)](/zh/docs/reference/config-api/apiserver-encryption.v1/)
+* [kubelet 配置 (v1alpha1)](/zh/docs/reference/config-api/kubelet-config.v1alpha1/) 和
+  [kubelet 配置 (v1beta1)](/zh/docs/reference/config-api/kubelet-config.v1beta1/)
+* [kubelet credential providers (v1alpha1)](/zh/docs/reference/config-api/kubelet-credentialprovider.v1alpha1/)
```
Comment (tengqm) [inline content/zh/docs/reference/_index.md:162]: 如果翻译的话，就都翻译了

```suggestion
* [kubelet 凭据驱动 (v1alpha1)](/zh/docs/reference/config-api/kubelet-credentialprovider.v1alpha1/)
```

**Pull request merged**